src/sql/parts/registeredServer/common/objectExplorerService.ts

```diff
@@ -65,16 +65,29 @@ export interface IObjectExplorerService {
 	onSelectionOrFocusChange: Event<void>;
 
 	getServerTreeView(): ServerTreeView;
+
+	getActiveConnections(): { connectionId: string, nodeInfo: sqlops.NodeInfo }[];
```

Comment:
Can you provide an interface for this object return type? It's standard in the VSCode APIs and makes life easier.